packages/jest-snapshot/src/__tests__/utils.test.ts

```diff
@@ -308,154 +302,151 @@ describe('removeLinesBeforeExternalMatcherTrap', () => {
 describe('DeepMerge with property matchers', () => {
   const matcher = expect.any(String);
 
-  /* eslint-disable sort-keys */
-  // to keep keys in numerical order rather than alphabetical
-  const cases = [
+  it.each(
```

Comment:
Ignore white space. Had to move it inline. For reasons see second example here: https://jestjs.io/docs/api#array-of-arrays